odoo_tools/conversion/convert_new_img.py

```diff
@@ -281,9 +281,52 @@ def remove_files():
             raise ValueError(f"unexpected file {name}. Is it a symlink?")
 
 
+def convert_env_lines(old_env_lines):
+    new_env_lines = []
+    for line in old_env_lines:
+        new_line = (
+            line.replace(
+                "/odoo/src/addons", "/odoo/src/odoo/odoo/addons, /odoo/src/odoo/addons"
+            )
+            .replace("/odoo/external-src/paid-modules", "/odoo/odoo/paid-modules")
+            .replace("/odoo/external-src/enterprise", "/odoo/src/enterprise")
+            .replace("/odoo/external-src", "/odoo/odoo/external-src")
+            .replace("/odoo/local-src", "/odoo/odoo/addons")
+        )
+        if "odoo-cloud-platform" in new_line:
+            continue
+        new_env_lines.append(new_line)
+    return new_env_lines
+
+
 def copy_dockerfile():
     shutil.move("odoo/Dockerfile", "Dockerfile.bak")
+    old_env_lines = []
+    with open("Dockerfile.bak") as old_dockerfile:
+        found_env = False
+        for line in old_dockerfile:
```

Comment:
Now pathlib is mandatory for this repo.

```suggestion
    old_dockerfile = Path("odoo/Dockerfile").rename("Dockerfile.bak")
    new_dockerfile = Path("Dockerfile")
    old_env_lines = []
    found_env = False
    with old_dockerfile.open() as fobj:
        for line in fobj:
```